tools/integration_tests/explicit_dir/explicit_dir_test.go

```diff
@@ -18,11 +18,18 @@ package explicit_dir_test
 import (
 	"testing"
 
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
 	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup/implicit_and_explicit_dir_setup"
 )
 
 func TestMain(m *testing.M) {
 	flags := [][]string{{"--implicit-dirs=false"}}
 
+	// Run tests for mountedDirectory only if --mountedDirectory and --testbucket flag is set.
+	setup.RunTestsForMountedDirectoryFlag(m)
+
+	// Run tests for testBucket only if --testbucket flag is set.
+	setup.SetUpTestDirForTestBucketFlag()
```

Comment:
what is the difference between this and below implcit_explicit.RunTests?

Reply:
This was done to test some changes. Forgot to revert. 
Reverted now.